exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java

```diff
@@ -258,6 +258,11 @@ public String getExecutionDuration() {
     return NOT_AVAILABLE_LABEL;
   }
 
+  //Threshold to be used by WebServer in issuing warning
+  public String getNoProgressWarningThreshold() {
+    return String.valueOf(FragmentWrapper.getProgressWarningThreshold());
```

Comment:
Again, please do not use static here, store DrillConfig if you need this threshold value.